packages/eds-tokens-build/src/index.ts

```diff
@@ -68,6 +71,48 @@ const lightDarkTransform: StyleDictionary.Transform = {
   },
 }
 
+//MARK: SpaceToggleTransform
+const densitySpaceToggleTransform: StyleDictionary.Transform = {
```

Comment:
Great work. Could we a create separate file for this transform and maybe also for the other complex transforms we have? I think that would clean up this file and make it more readable.

Reply:
We'll do this in a sepereate branch later so we keep the git history for these changes, plus there is no time